theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -71,5 +78,12 @@
                           class="glyphicon glyphicon-circle-arrow-left icon-button btn-edit"></span>
             </a>
         </div>
+
+        {% if show_manage_access %}
+            <a data-toggle="modal" data-target="#manage-access">
+                    <span data-toggle="tooltip" data-placement="auto" title="Manage who has access"
+                          class="glyphicon glyphicon-lock icon-button btn-edit"></span>
+            </a>
+        {% endif %}
```

Comment:
@mauriel133 This same {% if show_manage_access %} conditional block are included in both if and else block, which introduces duplicate code. I think you can simply move this out of if...else conditional block since it will be executed regardless of if condition.